tests/test_under_cached_property.py

```diff
@@ -1,15 +1,16 @@
 from types import ModuleType
+from typing import Any, Dict
 
 import pytest
 
 
 def test_under_cached_property(propcache_module: ModuleType) -> None:
     class A:
-        def __init__(self):
-            self._cache = {}
+        def __init__(self) -> None:
+            self._cache: Dict[str, Any] = {}
```

Comment:
FYI, things that compute as `Any` generate “uncovered” lines in the MyPy type coverage report. This is the recent gotcha I learned recently, looking deep into how it works, as it was rather unobvious initially.

Another gotcha is that such things can be caught by some checks and these checks may be disabled, but the fact that `Any` appears in types will contribute to the line becoming uncovered regardless.

That said, it seems like the keys are integers in tests, so it should be enough to use `int` instead...

```suggestion
            self._cache: Dict[str, int] = {}
```

Reply:
OTOH, looking into Codecov, it shows this as green. So maybe it's not really necessary here for some reason.